octopoes/octopoes/config/settings.py

```diff
@@ -89,3 +89,4 @@ def settings_customise_sources(
 DEFAULT_LIMIT = 50
 DEFAULT_OFFSET = 0
 QUEUE_NAME_OCTOPOES: str = "octopoes"
+GATHER_BIT_METRICS: bool = False
```

Comment:
The intention is to have a developer flag to toggle gathering [bit] metrics (without exposing it to the public). Please let me know what the best way of defining this would be.